src/angular-signature-pad.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+const angular = require('angular')
+const SignaturePad = require('signature_pad')
+
+BmSignaturePadController.$inject = ['$scope', '$element', '$window', '$log']
+
+function BmSignaturePadController ($scope, $element, $window, $log) {
+  if (!SignaturePad) {
+    $log.error('SignaturePad is required')
+    return
+  }
+  const vm = this
```

Comment:
Its because of this eslint rule: [controller-as-vm](https://github.com/Gillespie59/eslint-plugin-angular/blob/master/docs/controller-as-vm.md)